linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml

```diff
@@ -46,6 +48,10 @@ template:
     vars:
         option: timestamp_timeout
         variable_name: "var_sudo_timestamp_timeout"
+        # optional minus char added so remediation can detect properly if item is already configured
+        option_regex_suffix: '=[-]?\w+\b'
```

Comment:
This is fine, but just note that this regex only currently applies to the OVAL (which you've turned off below and replaced with a custom one above). :-)

Reply:
That's true. I overlooked this, it's kind of useless then. The quick fix is to update the regex directly to accept the minus char.

Reply:
actually it applies to the bash remediation as well: https://github.com/ComplianceAsCode/content/blob/master/shared/templates/sudo_defaults_option/template.py#L26 and without it the remediation doesn't work properly. I'll probably leave it like this.

Reply:
Ah ok :) No reason to hold this up then. :)